examples/graphene_apps/python_worker/README.md

```diff
@@ -107,7 +107,7 @@ https://creativecommons.org/licenses/by/4.0/
 
 - To run python worker as a docker container in Graphene-SGX environment and to use a test application to send work order requests, execute the following command from [python_worker](http://github.com/hyperledger/avalon/tree/master/examples/graphene_apps/python_worker) directory.
 
-  `docker-compose -f docker-compose.yaml -f docker-compose-graphene-sgx.yaml up`
+  `docker-compose -f docker-compose.yaml -f docker/compose/avalon-graphene-sgx.yaml up`
```

Comment:
Please don't change this. This is testing with out Avalon using test client.

Reply:
Updated.